packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -129,7 +143,7 @@ class IosPlatformImages {
   /// Returns null if the resource can't be found.
   ///
   /// See [https://developer.apple.com/documentation/foundation/nsbundle/1411540-urlforresource?language=objc]
-  static Future<String> resolveURL(String name, [String ext]) {
+  static Future<String?> resolveURL(String name, [String? ext]) {
```

Comment:
Since we're doing breaking changes anyway, and optional positional elements are strongly discouraged, should we take the opportunity to make this `{String? extension}`?

Reply:
+1